### DISCUSSION TITLE
Benutzung von HomeMaticSPVideoDoorBellAccessory

### DISCUSSION THREAD

**Discussion by NStiens** [thkl/hap-homematic#443] [Q&A] [answered]
Hallo zusammen,

ich würde gerne eine Kamera als HomeMaticSPVideoDoorBellAccessory im HomeKit anzeigen lassen.
Dabei stoße ich immer wieder auf ein paar Installation / Benutzung fragen.

1. Installation ffmpeg: Wie und wo? Leider hat man per SSH nur read-only Zugriff. Eventuell gibt es da eine gute Anleitung?
2. Wenn man ffmpeg auf den USB Stick kopiert und von dort aus startet, dann kommt leider "Permission denied"?
3. Lässt sich nach Installation ffmpeg testen? Auch wichtig um den RTSP Link von der Kamera zu testen.

Danke!

**Comment by ChrWi** [comment, marked as answer]
Hallo @NStiens, schau mal unter #226
